pkgs/development/python-modules/flatbuffers/default.nix

```diff
@@ -8,6 +8,10 @@ buildPythonPackage rec {
 
   sourceRoot = "source/python";
 
+  # flatbuffers needs VERSION environment variable for setting the correct
+  # version, otherwise it uses the current date.
+  VERSION = "${version}";
```

Comment:
Can we move this into preBuild or similar with an export?

Reply:
I think it's fine to use a variable here because this also works with nix-shell than to get a build environment.